src/fields/CoordinatesInput/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react'
+import styled from 'styled-components'
+
+import { CoordinatesFormat } from './constants'
+import { DDCoordinatesInput } from './DDCoordinatesInput'
+import { DMDCoordinatesInput } from './DMDCoordinatesInput'
+import { DMSCoordinatesInput } from './DMSCoordinatesInput'
+
+export type CoordinatesInputProps = {
+  coordinates: number[]
+  coordinatesFormat: CoordinatesFormat
+  onChange: (nextCoordinates: number[], coordinates: number[]) => void
```

Comment:
Just a detail it's nice to have a `Promisable<void>` on all function props to avoid any typing issue later on if the function is a promise.